Resources/Prototypes/Entities/Objects/Misc/candles.yml

```diff
@@ -32,7 +32,7 @@
       canResistFire: false
       alwaysCombustible: true
       canExtinguish: true
-      firestacksOnIgnite: 3.0
+      firestacksOnIgnite: 3.3
```

Comment:
Rationale for this change?

Reply:
All candles (not just the ones I made) don't completely burn, and the alternative is changing the thresholds for when they destruct or changing how fire heat damage is done which I wanted to avoid.